Doc/tools/extensions/availability.py

```diff
@@ -68,14 +69,17 @@ def run(self) -> list[nodes.container]:
             refwarn=True,
         )
         sep = nodes.Text(": ")
-        parsed, msgs = self.state.inline_text(self.arguments[0], self.lineno)
```

Comment:
It is somewhat surprising that this doesn't work, as `sphinx.util.nodes.extract_messages()` should pick up the rawsource attributes -- we shouldn't need to parse & reparse.

Reply:
Even when added `not WASI, not Android.` translation string by hand, it wasn't picked up in the build process.